#! /bin/sh -e
## 04_python_nexec.dpatch by Tilman Koschnick <til@subnetz.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove sha-bang line of python modules to avoid dependency on python.
## DP: Anything importing these modules will depend on python anyway.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/til/debian/gpsd/gpsd/gps.py gpsd/gps.py
--- /home/til/debian/gpsd/gpsd/gps.py	2004-10-16 12:25:55.000000000 +0200
+++ gpsd/gps.py	2004-11-28 23:13:58.589150234 +0100
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-#
 # gps.py -- Python interface to GPSD.
 #
 import time, socket, sys
